test/system/030-run.bats

```diff
@@ -85,4 +85,16 @@ echo $rand        |   0 | $rand
     run_podman 1 run --rm $IMAGE sh -c /bin/false
 }
 
+# 'run --rmi' deletes the image in the end unless it's used by another container.
+@test "podman run --rmi" {
+    run_podman pull redis
+    run_podman run redis /bin/true
+    run_podman images | grep redis
+    run_podman run --rmi --rm redis /bin/true
+    run_podman images | grep redis
+    run_podman 0 rm -a
+    run_podman 0 run --rmi --rm redis /bin/true
+    run_podman 1 image exists redis
+}
+
```

Comment:
It would be good to have a test that fails to remove the image due to it being in use by another container.

Reply:
Done! :+1: :smile_cat: